media/videostitcher/create_vod_session.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 Google LLC
+// Copyright 2024 Google LLC
```

Comment:
Don't update copyright dates on existing files, here and elsewhere.

Reply:
Done